srcpkgs/popt/template

```diff
@@ -1,16 +1,20 @@
 # Template file for 'popt'
 pkgname=popt
-version=1.16
-revision=7
+version=1.18
+revision=1
 build_style=gnu-configure
+hostmakedepends="autoconf tar automake libtool gettext-devel"
 short_desc="Command line option parsing library"
 maintainer="Orphaned <orphan@voidlinux.org>"
-homepage="http://rpm5.org"
 license="MIT"
-#distfiles="http://rpm5.org/files/$pkgname/$pkgname-$version.tar.gz"
-distfiles="https://fossies.org/linux/misc/popt-1.16.tar.gz"
-checksum=e728ed296fe9f069a0e005003c3d6b2dde3d9cad453422a10d6558616d304cc8
+homepage="https://rpm.org/"
+changelog="https://github.com/rpm-software-management/popt/releases/tag/popt-1.18-release"
```

Comment:
I'm not thrilled about this Github release page serving as the changelog, not least because it will get out of sync if somebody forgets to update the version and we generally don't want to replace the static version with `$version` or anything like that. Might be better just to drop the changelog until upstream updates the in-repo `CHANGES` file.

Reply:
Done!